starter-code/src/memory.js

```diff
@@ -1,15 +1,32 @@
-// var MemoryGame = function (cards) {
-//   this.cards = cards;
-// };
+var MemoryGame = function (cards) {
```

Comment:
Remember that you have to create an instance of the constructor to initialize the game